demonstrations/tutorial_classical_kernels.py

```diff
@@ -0,0 +1,738 @@
+=======================================================================
+
+.. meta::
+    :property="og:description": Finding a QK to approximate the Gaussian
+    kernel.
+    :property="og:image": https://pennylane.ai/qml/_images/toy_qek.png
+
+.. related::
+
+   tutorial_kernels_module Training and evaluating quantum kernels
+   tutorial_kernel_based_training Kernel-based training of quantum models with
+   scikit-learn
+   tutorial_expressivity_fourier_series Quantum models as Fourier series
+
+
+*Author: Elies Gil-Fuster (Xanadu resident). Posted: DD MMMM 2021.*
+
+Forget about advantages, supremacies, or speed-ups.
+Let us understand better what we can and cannot do with a quantum computer.
+More specifically, in this demo we want to look into quantum kernels and ask
+whether we can replicate classical kernel functions with a quantum computer.
+Lots of researchers have lengthily stared at the opposite question, namely that
+of classical simulation of quantum algorithms.
+Yet, by studying what classes of functions we can realize with quantum kernels,
+we can gain some insight into their inner workings.
+
+Usually, in quantum machine learning (QML) we use parametrized quantum circuits
+(PQCs) to find good functions, whatever *good* means here.
+Consequently, since kernels are just one specific kind of well-defined
+functions, the task of finding a quantum kernel (QK) that approximates a given
+classical one could be posed as a supervised learning regression problem.
+The way to attack this task would be to define a loss function directly
+quantifying the distance between both functions (the classical kernel function,
+and the PQC-based hypothesis).
+This sort of approach is more or less oblivious to the Fourier representation
+of the kernels, though.
+
+We want to study the link between classical and quantum kernels through the
+lens of the Fourier representation of a kernel, which is a common tool in
+classical ML.
+Two functions can only have the same Fourier spectrum if they are the same
+function, and it turns out that for certain classes of quantum circuits `we can
+theoretically describe the Fourier spectrum rather well
+<https://pennylane.ai/qml/demos/tutorial_expressivity_fourier_series.html>`_.
+
+Using this theory together with some good old-fashioned convex optimisation, we
+will derive a quantum circuit that approximates the famous Gaussian kernel.
+
+In order to keep the demo short and sweet, we focus on one simple example, but
+the same ideas would apply for more general scenarios.
+Also, Refs. [#QEK]_, [#Fourier]_, and [#qkernels]_ should be helpful for those
+who'd like to see the underlying theory of QKs (and also Quantum Embedding
+Kernels) and their Fourier representation.
+So tag along if you'd like to see how we build a quantum kernel that
+approximates the well-known Gaussian kernel function!
+
+|
+
+.. figure:: ../demonstrations/classical_kernels/QEK.jpg
+    :align: center
+    :width: 80%
```

Comment:
```suggestion
    :width: 70%
```
Or even smaller? I always like pictures to be scaled such that the text they contain is of the same size as the normal text... [Minor one]